images/manageiq-base/Dockerfile

```diff
@@ -9,10 +9,13 @@ ARG MIQ_ORG=ManageIQ
 ARG SUI_ORG=ManageIQ
 ARG APPLIANCE_ORG=ManageIQ
 
+ARG ARCH=amd64
```

Comment:
I think I would prefer the default to be `x86_64` now that that's available in the new dumbinit release.

Reply:
okay